pkg/cluster/compute/affinityrule/affinityrule.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package affinityrule
+
+import (
+	"github.com/microsoft/wmi/pkg/base/host"
+	"github.com/microsoft/wmi/pkg/base/instance"
+	"github.com/microsoft/wmi/pkg/base/query"
+	"github.com/microsoft/wmi/pkg/constant"
+	wmi "github.com/microsoft/wmi/pkg/wmiinstance"
+	fc "github.com/microsoft/wmi/server2019/root/mscluster"
```

Comment:
Just for my own understanding, I noticed we have pkgs `wmi/server2019` and `wmi/server23h2`. Is there a reason server23h2 is referenced so much less frequently? What's the difference?